### DISCUSSION TITLE
Manifest unknown

### DISCUSSION THREAD

**Discussion by everydaylearner** [ejsuncy/sense_energy_prometheus_exporter#49] [General]
Hi There,

I am trying to pull the image but getting following message; 

"error: initializing source docker://ghcr.io/ejsuncy/sense_energy_prometheus_exporter:0.7.0a: reading manifest 0.7.0a in ghcr.io/ejsuncy/sense_energy_prometheus_exporter: manifest unknown"

is the container version and path is correct? 

Thank you!

**Comment by ejsuncy** [comment]
Sorry, that's my bad. I'll build an image for the `main` branch this evening. In the meantime you can switch to the `0.6.x` branch and follow those readme instructions:
```
docker pull ghcr.io/ejsuncy/sense_energy_prometheus_exporter:0.6.1
```

**Reply by ejsuncy** [reply]
I've built and pushed an image for the `main` branch with the `0.7.0a` tag, so the following command now works:
```
docker pull ghcr.io/ejsuncy/sense_energy_prometheus_exporter:0.7.0a
```
